### PR TITLE
virtcontainer : sconstification of constifiable vars

### LINKED ISSUE #142: constification of constifiable vars
_From @egernst on February 22, 2018 23:22_

Noticed that we pretty liberally use var ( ) for what could in many cases be treated as a const.  
Example can be seen at https://github.com/containers/virtcontainers/blob/master/kata_agent.go#L38

Some of these are more explicitly marked as var since they are updated during unit tests (see *_test.go) to aid in testing.  For the remainder it'd be a good idea to mark these as consts.

_Copied from original issue: containers/virtcontainers#636_

### PR DESCRIPTION
**Pull request by jiulongzaitian**
virtcontainer : sconstification of constifiable vars

Fixes #142

Signed-off-by: zhangjie <zhangjie0619@yeah.net>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *.patch
 *.o
 *.swp
+.idea/
```
Comment (gnawux) [inline .gitignore]: please remove the `.gitignore` update from this commit. others looks good

Comment (jiulongzaitian) [reply .gitignore]: ok

**Comment by WeiZhang555** [comment]
LGTM.
Waiting CI

**Comment by jodh-intel** [comment]
This is the sort of PR I like! :smile:

lgtm

**Comment by jodh-intel** [comment]
F27 CI build restarted...

**Comment by gnawux** [comment]
> `  Status code: 503 for https://mirrors.fedoraproject.org/metalink?repo=updates-released-f27&arch=x86_64`

looks F27 built failed by some fedora service failure.

**Comment by WeiZhang555** [comment]
Since jenkins-ci-ubuntu-16-04 passed and we have enough LGTMs, I'll merge this. I think the CI failure is caused by unstable tests as this PR only modifies var to const.

Thanks @jiulongzaitian for your contribution!

**Pull request merged**